test-cases/performance/perf-regression-throughput-125gb-with-ebs-gp3-volume.yaml

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Where is the prepare cmd command here?
The read and mixed can't run without it.

Reply:
As we discussed, now we use the combination of several test config. Prepare command defined in base performance config.yaml:
perf-regression-throughput-125gb-with-ebs-gp3-volume.jenkinsfile:
- test-cases/performance/perf-regression-throughput-125gb.yaml
-  test-cases/performance/perf-regression-throughput-125gb-with-ebs-gp3-volume.yaml

The write, read, mixed command were redefined because number of threads is different.

Reply:
If it's the case, it means we should have here only the gp3 parameters and user-prefix.
The rest should be exactly like 125gb-with-ebs.

Reply:
The names of the configuration should refelct exactly that, so it's configuration/ebs-gp3-4tb-16k-iops

Reply:
fixed